brion/blueConfig.h

```diff
@@ -92,6 +92,32 @@ class BlueConfig : public boost::noncopyable
     }
     //@}
 
+    /** @name Semantic read API */
+    //@{
+    /** @return the full path to the circuit.mvd2 file. @sa Circuit */
```

Comment:
in the light of mvd3 (and potentially ZeroEQ-accessible circuits), shouldn't this be 'the URI to the circuit information'?